omaha/core/scheduler_unittest.cc

```diff
@@ -0,0 +1,180 @@
+// Copyright 2007-2019 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ========================================================================
+
+#include "omaha/core/scheduler.h"
+
+#include "omaha/base/constants.h"
+#include "omaha/base/highres_timer-win32.h"
+#include "omaha/testing/unit_test.h"
+#include "omaha/third_party/smartany/scoped_any.h"
+
+namespace omaha {
+
+namespace {
+
+inline void ASSERT_SIGNALLED_BEFORE(HANDLE handle, DWORD timeout_ms) {

```

Comment:
Since these are functions, we should follow the naming convention for functions.

Reply:
Done